contracts/Refunder.sol

```diff
@@ -39,11 +39,17 @@ contract Refunder is
     uint256 public REFUND_OP_GAS_COST = 5106;
 
     struct Refundable {
```

Comment:
I think that it would be better if we keep the inline docs consistent and do it the following way:

```Solidity
/**
 * @notice Struct storing the refundable data for a given target
 * isSupported marks whether the refundable is supported or not
 * validationContract contract address to call for business logic validation on every refund
 * validationFunc function to call for business logic validation on every refund
 */
struct Refundable {
...

}


```